nginx_proxy/rootfs/etc/s6-overlay/s6-rc.d/nginx/run

```diff
@@ -56,6 +56,23 @@ sed -i "s/%%HA_PORT%%/$HA_PORT/g" /etc/nginx.conf
 [ -n "$HSTS" ] && HSTS="add_header Strict-Transport-Security \"$HSTS\" always;"
 sed -i "s/%%HSTS%%/$HSTS/g" /etc/nginx.conf
 
+if bashio::config.true 'proxy_protocol.enabled'; then
+    REAL_IP_FROM=$(bashio::config 'proxy_protocol.real_ip_from')
+    if [ -z "$REAL_IP_FROM" ]; then
+        echo "real_ip_from under proxy_protocol has to be provided"
+        exit 1
+    fi
+    sed -i "s#%%REAL_IP_FROM%%#$REAL_IP_FROM#g" /etc/nginx/serverProxyProtocol.conf
```

Comment:
It seems that camel case is not common for nginx config files. Also, includes are typically stored in `conf.d` sub directory (see https://docs.nginx.com/nginx/admin-guide/basic-functionality/managing-configuration-files/). Can you rename these include files and move them to `conf.d` directory?